src/Control/Email/Email.php

```diff
@@ -933,23 +967,29 @@ public function findPlainPart()
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::getTextBody()
+     *
      * @return bool
      */
     public function hasPlainPart()
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::getTextBody()');
         if ($this->getSwiftMessage()->getContentType() === 'text/plain') {
             return true;
         }
         return (bool) $this->findPlainPart();
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::getTextBody()
+     *
      * Automatically adds a plain part to the email generated from the current Body
      *
      * @return $this
      */
     public function generatePlainPartFromBody()
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::getTextBody()');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Use getTextBody() instead');
```

Reply:
Is that correct? Does that actually _generate_ the plain part? Or would it be more correct to say there's no equivalent of this method?

Reply:
Should say replace with not equivalent as discussed above.

Reply:
Updated